src/components/DetailComment/DetailComment.stories.ts

```diff
@@ -0,0 +1,11 @@
+import DetailComment from './DetailComment.vue'
+
+export default {
+  title: 'component/DetailComment',
+  component: DetailComment
+}
+
+export const Default = () => ({
+  components: { DetailComment },
+  template: '<detail-comment />'
+})
```

Comment:
It would be nice to have some use cases in here.
This will be done in the future when we refactor this component (other ticket). FYI.

Reply:
examples are in src/stories folder

Reply:
Is there an example for DetailComment?

Reply:
do you mean the available props and events?

Reply:
Yes.

Reply:
![image](https://user-images.githubusercontent.com/64869009/92178705-fc128a80-edf7-11ea-9016-89ea48b68153.png)

Reply:
you can see that in screen like this

Reply:
OK for now.